service/README.md

```diff
@@ -145,6 +145,19 @@ The file at `<path-to-shards-file>` should be a `.json` file containing a list o
 
 ---
 
+## Running tests
+Set these environment variables:
+
+- `TEST_DEPLOYER_PRIVATE_KEY`: the private key of the deployer - logged in the node's logs (private key of Account #0)
+- `TEST_CONTRACTS_DIRECTORY`: absolute path to the directory containing the Sarcophagus Contracts project (https://github.com/sarcophagus-org/sarcophagus-v2-contracts/)
+- `TEST_NETWORK_STARTUP_TIME`: Set if needed. This sets a buffer in milliseconds to wait for the network to start and Sarco contracts to finish deploying. Defaults to 6000.
```

Comment:
should these new env vars get added to the `.env.example` file?

Reply:
I did add them initially but later decided against it. Reasoning is that I didn't want to add potential confusion to archaeologist users who are setting up their nodes (ie, "uhm what value do I set this to...?").
As the only people who will care about tests would (or _should_) read the readme anyway, there's really nothing lost there.

I'm open to being convinced otherwise though.

Reply:
@DarksightKellar -- that's a great point, agree with you that they shouldn't go in there.

Maybe we set up a separate env file for testing, like `.env.test`? I'm trying to think through this from perspective of a developer on the project. We don't need to solve this now, but something to think about.